guides/common/modules/con_how-puppet-integrates-with-project.adoc

```diff
@@ -49,13 +35,17 @@ by syncing repositories in custom products
 endif::[]
 and by using {ContentManagementDocURL}Managing_Activation_Keys_content-management[activation keys] and {ContentManagementDocURL}Managing_Content_Views_content-management[content views].
 endif::[]
-. Install Puppet agent on hosts during xref:Installing_and_Configuring_Puppet_Agent_during_Host_Provisioning_{context}[provisioning], xref:installing-and-configuring-puppet-agent-during-host-registration_{context}[registration], xref:Installing_and_Configuring_Puppet_Agent_Manually_{context}[manually], or by remote job execution.
+. Install Puppet agent on hosts during xref:common/modules/proc_installing-and-configuring-puppet-agent-manually.adoc#Installing_and_Configuring_Puppet_Agent_during_Host_Provisioning_{context}[provisioning], xref:common/modules/proc_installing-and-configuring-puppet-agent-during-host-registration.adoc#installing-and-configuring-puppet-agent-during-host-registration_{context}[registration], xref:common/modules/proc_installing-and-configuring-puppet-agent-manually.adoc#Installing_and_Configuring_Puppet_Agent_Manually_{context}[manually], or by remote job execution.
 
-.Additional resources
-ifdef::katello,satellite,orcharhino[]
-* {ContentManagementDocURL}[{ContentManagementDocTitle}]
-endif::[]
-* {ManagingHostsDocURL}registering-hosts-and-setting-up-host-integration_managing-hosts[Registering hosts and setting up host integration] in _{ManagingHostsDocTitle}_
-* {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote_Jobs_managing-hosts[Configuring and setting up remote jobs] in _{ManagingHostsDocTitle}_
```

Comment:
I decided to remove these additional resources because I think they are quite generic: It's a very high-level section and I don't consider it likely that users will feel the need to jump to the details of how to configure a remote job.